tests/Handler/ExceptionHandlerTest.php

```diff
@@ -82,3 +93,11 @@ public function testMethodNotAllowed()
         $this->assertEquals('GET,PUT', $response->getHeaderLine('Allow'));
     }
 }
+
+class ExceptionWithHttpStatusCode extends \Exception
```

Comment:
I would put this in a separate class. Having it here violates the requirement of "one class per file".